requirements.txt

```diff
@@ -7,3 +7,4 @@ pytest
 -e ./zulip
 -e ./zulip_bots
 -e ./zulip_botserver
+-e git+https://github.com/zulip/zulint@aaed679f1ad38b230090eadd3870b7682500f60c#egg=zulint==0.0.1
```

Comment:
`tools/provision` now warns: `WARNING: Requested zulint==0.0.1 from git+https://github.com/zulip/zulint@aaed679f1ad38b230090eadd3870b7682500f60c#egg=zulint==0.0.1 (from -r /home/anders/zulip/python-zulip-api/requirements.txt (line 10)), but installing version 1.0.0`.

(We have a bunch of these in zulip/zulip too. I was planning to clean them up soon.)

Reply:
Done!